security/rbac/delta_configs/connect-avro-distributed.properties.delta

```diff
@@ -53,10 +35,13 @@ confluent.metadata.http.auth.credentials.provider=BASIC
 
 config.providers=secret
 config.providers.secret.class=io.confluent.connect.secretregistry.rbac.config.provider.InternalSecretConfigProvider
+config.providers.secret.param.master.encryption.key=password1234
 
-config.providers.secret.param.master.encryption.key=<encryption key>
+config.providers.secret.param.kafkastore.topic=_secrets
 config.providers.secret.param.kafkastore.bootstrap.servers=localhost:9092
-# Can also be SASL_SSL
-config.providers.secret.param.kafkastore.security.protocol=SASL_PLAINTEXT config.providers.secret.param.kafkastore.sasl.mechanism=OAUTHBEARER
+config.providers.secret.param.kafkastore.security.protocol=SASL_PLAINTEXT
+config.providers.secret.param.kafkastore.sasl.mechanism=OAUTHBEARER
 config.providers.secret.param.kafkastore.sasl.login.callback.handler.class=io.confluent.kafka.clients.plugins.auth.token.TokenUserLoginCallbackHandler
 config.providers.secret.param.kafkastore.sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required username="connect" password="connect1" metadataServerUrls="http://localhost:8090";
+
+rest.advertised.host.name=localhost
```

Comment:
OOC, do things break if this isn't specified?

Reply:
Nope!  I'll cut it.